### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by garanews**
Parsed all repo with a regexp of common misspelled words and fixed.

### CODE REVIEW

File: promql/engine.go [hunk 1]
```diff
@@ -1500,7 +1500,7 @@ func (ev *evaluator) matrixIterSlice(it *storage.BufferedSeriesIterator, mint, m
 			ev.currentSamples++
 		}
 	}
-	// The seeked sample might also be in the range.
+	// The sought sample might also be in the range.
```
Comment (brian-brazil) [inline promql/engine.go:1503]: This was right.

File: tsdb/querier_test.go [hunk 2]
```diff
@@ -867,7 +867,7 @@ func TestPopulateWithDelSeriesIterator_DoubleSeek(t *testing.T) {
 	testutil.Equals(t, float64(2), v)
 }
 
-// Regression when seeked chunks were still found via binary search and we always
+// Regression when sought chunks were still found via binary search and we always
```
Comment (brian-brazil) [inline tsdb/querier_test.go:870]: This was right.

File: vendor/github.com/Azure/go-autorest/CHANGELOG.md [hunk 3]
```diff
@@ -785,7 +785,7 @@ longer work with golint.
 
 ## v9.0.0
 
-> **IMPORTANT:** This release was intially labeled incorrectly as `v8.4.0`. From the time it was released, it should have been marked `v9.0.0` because it contains breaking changes to the MSI packages. We appologize for any inconvenience this causes.
+> **IMPORTANT:** This release was initially labeled incorrectly as `v8.4.0`. From the time it was released, it should have been marked `v9.0.0` because it contains breaking changes to the MSI packages. We appologize for any inconvenience this causes.
```
Comment (brian-brazil) [inline vendor/github.com/Azure/go-autorest/CHANGELOG.md:788]: Don't change vendored packages, any such changes should be done upstream.

Comment (garanews) [reply vendor/github.com/Azure/go-autorest/CHANGELOG.md]: done

File: promql/engine.go (hunk 1 above)
Comment (garanews) [reply promql/engine.go:1503]: sure? https://www.oxfordlearnersdictionaries.com/definition/english/seek

File: tsdb/querier_test.go (hunk 2 above)
Comment (garanews) [reply tsdb/querier_test.go:870]: sure? https://www.oxfordlearnersdictionaries.com/definition/english/seek

File: promql/engine.go (hunk 1 above)
Comment (brian-brazil) [reply promql/engine.go:1503]: Yes, the method is called Seek and I've never heard of sought time for a hard drive.

Comment (garanews) [reply promql/engine.go:1503]: Me neither, double checked on dictionary! (english is not my native language)
Follow your instructions...

**Comment by brian-brazil** [comment]
This PR is still changing vendored files, and lacks a DCO.

**Comment by garanews** [comment]
Signed-off-by: garanews <puntogtg@tiscali.it>

**Comment by garanews** [comment]
@brian-brazil having some issue with dco, can you help me?

**Comment by brian-brazil** [comment]
Squashing the commits is generally easiest way to handle it.

You're still touching a vendored file in the web dir.

**Comment by garanews** [comment]
After your suggestion I preferred create a new PR, going to close this one.

**Comment by ArthurSens** [comment]
Hi @garanews 

It looks like you are having some trouble with git, DCO, and with golang vendors.

DCO is to make sure all your commits pass the legal requirements to be merged. You can see the legal requirements [here](https://developercertificate.org/). It's really simple to make the DCO test to pass, you just need to add your signature at the end of all commit messages. You can edit you commit messages with `git rebase`

Looks like you've added several commits to this PR(17 so far), some of those are indeed yours and others look like they were added unintentionally. What I suggest you is to run `git rebase -i HEAD~17`, git will open a text editor for you to manage all the 17 commits. The commits are ordered and follow this logic: 
```
verb   commit identifier    commit message
```

Read the commands written on that text editor, maybe google some if you are not sure what to do with the commits. You probably want to `drop`, `reword`, `squash` and `pick` some commits.

After rebasing, you can push your changes with `git push --force`

For future commits, you can create them with the `--signoff` flag, that will add your signature automatically: `git commit -s -m "some message"`

**Comment by garanews** [comment]
For these reasons I created a new pr #8023 more clear and closed this one.
1) didn't know about dco, I see it is easy, if you do at the beginning, and not after 10 commits, when other people are committing too
2) didn't know about vendor folder, don't know how your project is structured so after fixing these files had to revert back.

I tried to fix all of these things with commands you suggested me, without success and pr was not looking good, so created new one.

**Pull request closed without merging**